mobile/ios/EntePeopleWidget/EntePeopleWidget.swift

```diff
@@ -0,0 +1,276 @@
+//
+//  EntePeopleWidget.swift
+//  EntePeopleWidget
+//
+//  Created by Prateek Sunal on 5/15/25.
+//  Copyright © 2025 The Chromium Authors. All rights reserved.
```

Comment:
Can you configure your IDE to remove these?

Reply:
This is created whenever a new WidgetExtension is added from XCode. Will remove them manually.